qiskit/circuit/quantumcircuit.py

```diff
@@ -844,10 +845,15 @@ def compose(
             QuantumCircuit: the composed circuit (returns None if inplace==True).
 
         Raises:
-            CircuitError: if composing on the front.
-            QiskitError: if ``other`` is wider or there are duplicate edge mappings.
+            CircuitError: if no correct wire mapping can be made between the two circuits, such as
+                if ``other`` is wider than ``self``.
+            CircuitError: if trying to emit a new circuit while ``self`` has a partially built
+                control-flow context active, such as the context-manager forms of :meth:`if_test`,
+                :meth:`for_loop` and :meth:`while_loop`.
+            CircuitError: if trying to compose to the front of a circuit when a control-flow builder
+                block is active; there is no clear meaning to this action.
 
-        Examples::
+        Examples:
 
             lhs.compose(rhs, qubits=[3, 2], inplace=True)
```

Comment:
How does this get rendered now that it's not part of a code block anymore?

Reply:
Ah, I made a bit of a mess of it.  The current docs render wrong because the `.. parsed-literal` directive is included in the output (since it's in a codeblock), so my change fixed that, but I forgot to make the actual code a code block again.  Thanks, fixed in c358039.